src/commands/generate.ts

```diff
@@ -0,0 +1,161 @@
+import { Flags, CliUx } from '@oclif/core';
+import Command from '../base';
+// eslint-disable-next-line
+// @ts-ignore
+import AsyncAPIGenerator from '@asyncapi/generator';
+import path from 'path';
+import os from 'os';
+import { load } from '../models/SpecificationFile';
+import { Example } from '@oclif/core/lib/interfaces';
+
+export class GenerateFlagParser {
```

Comment:
Please avoid creating helper classes, that you can "merge" to the command class. Please move `parse` method to the `Generator` class and call it as `parseFlags` and return from it object as:

```ts
interface ParsedFlags {
  params: Record<string, string>;
  disableHooks: Record<string, string>;
  mapBaseUrlToFolder: {
    url: string;
    folder: string;
  }
}
```

Reply:
I suggested you to remove that class and move all methods (as private) to the `Generate` class :)